src/com/magento/idea/magento2uct/inspections/php/InheritedInterfaceInspection.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+package com.magento.idea.magento2uct.inspections.php;
+
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementVisitor;
+import com.jetbrains.php.lang.inspections.PhpInspection;
+import com.jetbrains.php.lang.psi.elements.ClassReference;
+import com.jetbrains.php.lang.psi.elements.PhpClass;
+import com.jetbrains.php.lang.psi.visitors.PhpElementVisitor;
+import com.magento.idea.magento2uct.packages.IssueSeverityLevel;
+import com.magento.idea.magento2uct.settings.UctSettingsService;
+import org.jetbrains.annotations.NotNull;
+
+public abstract class InheritedInterfaceInspection extends PhpInspection {
+
+    @Override
+    public @NotNull PsiElementVisitor buildVisitor(
+            final @NotNull ProblemsHolder problemsHolder,
+            final boolean isOnTheFly
+    ) {
+        return new PhpElementVisitor() {
+
+            @Override
+            public void visitPhpClass(final PhpClass clazz) {
```

Comment:
Typo here

```suggestion
            public void visitPhpClass(final PhpClass class) {
```

Reply:
Hello, @eduard13! It is not a typo...class is a reserved word. It used as an alias in the java codebase.